packages/flutter/lib/src/material/bottom_navigation_bar.dart

```diff
@@ -154,6 +154,9 @@ class BottomNavigationBar extends StatefulWidget {
   /// [selectedIconTheme] and [unselectedIconTheme], and both
   /// [IconThemeData.color] and [IconThemeData.size] must be set.
   ///
+  /// [IconThemeData]s take precedence over [iconSize], [selectedItemColor]
+  /// and [unselectedItemColor] for icons.
+  ///
```

Comment:
```suggestion
  /// [selectedIconTheme] values will be used instead of [iconSize] and [selectedItemColor] for selected icons.
  /// [unselectedIconTheme] values will be used instead of [iconSize] and [unselectedItemColor] for unselected icons.
  ///
```

Reply:
And isn't this only true if `useLegacyColorScheme` is false?

Reply:
correct, I should mention this in the docs right?